README.md

```diff
@@ -79,8 +67,6 @@ Installs the Datadog agent on the target system, sets the API key, and start the
 **Notes for Windows**:
 
 * With Chef >= 12.6 _and_ the `windows` cookbook >= 1.39.0, Agent upgrades are known to fail.
-  For Chef>=12.6 users on Windows, we recommend pinning the `windows` cookbook to a lower version (`~> 1.38.0` for instance).
-
   If that's not an option, a known workaround is to use the `remove-dd-agent` recipe (since the `2.5.0` version of the present cookbook) to uninstall the Agent
```

Comment:
@remeh this sentence doesn't make sense anymore since you removed the previous one